frontend/packages/kubevirt-plugin/src/components/vm-templates/instantiate-template/TemplateForm.tsx

```diff
@@ -54,6 +61,23 @@ export const TemplateForm: React.FC<TemplateFormProps> = ({
   };
 
   const [parameters, setParameters] = React.useState<{}>(getParameterValues());
+
+  const missingParams = Object.entries(parameters).filter(
+    ([, value]) => !value || _.isEmpty(value),
+  );
+  const initValues = getParameterValues();
```

Comment:
useMemo, also isnt it the same as line 63 initial value?

Reply:
same :)